client/src/components/collectionPanel/CollectionSettingsTab/CollectionSettingsTab.tsx

```diff
@@ -0,0 +1,21 @@
+import GroupsInput from '../../groupsInput';
+import styles from './CollectionSettingsTab.module.css';
+
+type CollectionSettingsTabProps = {
+  groups: string[];
+  setGroups: (groups: string[]) => void;
+};
+
+export default function CollectionSettingsTab({
+  groups,
+  setGroups,
+}: CollectionSettingsTabProps) {
+  return (
+    <div className={styles.grid}>
+      <div className={styles.groupsLabel}>
+        <div>Groups</div>
```

Comment:
this extra div is not needed now that you are not using tables anymore.

Reply:
I still need it, because i want to align the label to the input field, and not the entire GroupInput component.
`.groupsLabel {
  align-self: start;
}

.groupsLabel > div {
  display: flex;
  align-items: center;
  height: 50px;
}`

Looks cleaner like that in my opinion.
First image using the described alignment, second without.

<img width="1017" alt="Screenshot 2023-12-02 at 17 27 15" src="https://github.com/EsperoTech/yaade/assets/53432978/394d67fc-e89a-4f45-9cf8-a437aad178ac">

<img width="1017" alt="Screenshot 2023-12-02 at 17 27 26" src="https://github.com/EsperoTech/yaade/assets/53432978/eabfc2b2-d878-4113-a350-5336c59db0e7">